idaloader.cpp

```diff
@@ -91,6 +125,12 @@ void pe_add_sections(XEXFile& file)
         continue;
       if (!strcmp(name, ".reloc"))
         continue;
+      //----------------------------------------------------------------------
+      // .idata section is bugging out in a few XEXs I tried with
+      // don't care to debug why, just move on
+      if (!strcmp(name, ".idata"))
```

Comment:
Bad reason to skip this. At the very least, this shouldn't be in this PR. It's been a long time since my x360 reversing days and I'm not sure what the idata section is used for, but a blanket continue; is reckless